buildSrc/src/main/groovy/nva.nvi.java-conventions.gradle

```diff
@@ -21,7 +21,7 @@ java {
     targetCompatibility = JavaVersion.VERSION_21 // bytecode target version
 }
 
-tasks.named('test') {
+tasks.named('test', Test) {
```

Comment:
This is an unrelated change to fix an IDE warning (as IntelliJ can now understand what type this is).